zkbind/src/org/zkoss/bind/impl/AbstractAnnotatedMethodInvoker.java

```diff
@@ -167,7 +167,11 @@ public void invokeMethod(Binder binder, Map<String, Object> bindingArgs, Event t
 					}
 
 					//super first
```

Comment:
You can move this comment, and note `@Init` is super first, `@Destory` is super last